py4DSTEM/process/phase/iterative_mixedstate_ptychography.py

```diff
@@ -2245,6 +2254,7 @@ def visualize(
         plot_convergence: bool = True,
```

Comment:
`iterations_grid` should be `Optional[Tuple[int, int]] = None` implicit optional may/will cause errors in the future with linters